refinery/ui/source/js/tool-launch/directives/tool-params.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Tool Params Component
+ * @namespace rpToolParams
+ * @desc Component for the parameter's panel.
+ * @memberOf refineryApp.refineryToolLaunch
+ */
+(function () {
+  'use strict';
+  angular
+    .module('refineryToolLaunch')
+    .component('rpToolParams', {
+      controller: 'ToolParamsCtrl',
+      require: {
+        displayCtrl: '^rpToolDisplay'
+      },
+      templateUrl: ['$window', function ($window) {
+        return $window.getStaticUrl('partials/tool-launch/partials/tool-params.html');
+      }]
```

Comment:
A plain string would work here. A few weeks ago Ilya added a method to generate the static url based on django settings. So we use that method to generate the static urls as seen above.